browsertests/browser_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
can this be kept inside the internal package, possibly within `internal/driver/browsertests` or something similar?

Reply:
> Plus internal/ means something special to Go, so placing a module crossing boundary inside an internal/... subtree may cause some confusion.

It does function properly to refer *up* the tree from an internal/... module (experiment at HEAD of https://github.com/liggitt/pprof/commits/chromedpdep/), and I thought putting this test-only module underneath `internal` would ensure nothing outside pprof could accidentally refer to it, which could reduce confusion.

But if you think it's less confusing as a visible test-only package, that's fine as well.